Update julia.md (#2463)
travis-ci/docs-travis-ci-com: user/languages/julia.md

Travis CI support for Julia is contributed by the community and may be removed
or altered at any time. If you run into any problems, please report them in the
[Travis CI Julia Community Forums](https://travis-ci.community/c/languages/julia)
and cc [@ararslan](https://github.com/ararslan), [@staticfloat](https://github.com/staticfloat), and [@StefanKarpinski](https://github.com/StefanKarpinski).

> General support for Julia on Travis CI can be found on [Travis Community](https://travis-ci.community/c/languages/julia) or on the [Julia Lang Slack Channel](https://julialang.slack.com) under the __#testing__ channel.

Reviewer: ```suggestion
For general Julia support on Travis CI go to the [Travis Community](https://travis-ci.community/c/languages/julia) or [Julia Lang Slack Channel](https://julialang.slack.com) in the __#testing__ channel.
```

it is possible to specify a custom dependency installation command as described
in the [general build configuration](/user/customizing-the-build/) guide.

> In a rare case, you may need to clone a private repo if it is a depenedancy of the repo you are trying to test. To add a private repo, check out the link here: [Private Dependencies](/user/private-dependencies/).  Once you have the repo added, you will need to copy it to your julia folder and then run the default build script.  Check out the script below for __Linux__ to see how that is done: 

Reviewer: ```suggestion
In a rare case, you may need to clone a private repo, if you are trying to test a dependency of the repo. To add a private repo, check out the link here: [Private Dependencies](/user/private-dependencies/).  Once you have the repo added, you will need to copy it to your julia folder and then run the default build script.  Check out the script below for __Linux__ to see how that is done: 
```

Author: @Pezi777 In this example, you are not testing a dependancy. You are testing the repo!

Author: Therefore I stand by the original language I used.

Author: @Pezi777 any thoughts?

Reviewer: we can keep the wording, but it is nevertheless "dependency" and not "depenedancy" - right?

Author: That’s correct! My mistake on the spelling error.
